jsonnet/kube-prometheus/ksm-autoscaler/ksm-autoscaler.libsonnet

```diff
@@ -3,11 +3,11 @@ local k = import 'github.com/ksonnet/ksonnet-lib/ksonnet.beta.4/k.libsonnet';
 {
```

Comment:
Changes in this file are also included in https://github.com/prometheus-operator/kube-prometheus/pull/769

Reply:
Oopsie. I'll fix it.

Reply:
Fixed :)